recipes/groff/build.sh

```diff
@@ -0,0 +1,15 @@
+set -x
+
+autoreconf -vfi
+./configure --prefix=$PREFIX
+
+# Workaround for long shebang lines
+find $SRC_DIR -type f | \
+    xargs -L1 perl -i.bak \
+        -pe 's,^#!\@PERL\@ -w,#!/usr/bin/env perl,;' \
+        -pe "s,perl -w,perl,;" \
+        -pe "s,$PREFIX/bin/perl,/usr/bin/env perl,;"
+
+# Workarond for dependency issue
+make -j${CPU_COUNT} font/devpdf/build_font_files
```

Comment:
Don't we have fonts elsewhere in conda-forge?

Reply:
I have added a comment to describe the actual need for this split make step.

Reply:
Usually we put fonts in separate packages / outputs, though, so then can be reused. Does that not apply here for some reason?

Reply:
All the font related stuff is packaged under <pkg-name>-<pkg-ver> directories: 
```
share/doc/groff-1.22.4/examples
share/groff/1.22.4/font
share/groff/1.22.4/oldfont
share/man/man5/groff_font.5
share/man/man5/lj4_font.5
```

I am not sure if they are meant to be shared. I didn't even find Fedora doing it:
https://fedora.pkgs.org/31/fedora-x86_64/groff-perl-1.22.3-20.fc31.x86_64.rpm.html

Reply:
Ok thanks!